app/frontend/src/locales/es/translation.json

```diff
@@ -154,6 +161,8 @@
         "suggestFollowupQuestions": "Pide al LLM que sugiera preguntas de seguimiento basándose en la consulta del usuario.",
         "textEmbeddings": "Cuando se selecciona, la búsqueda utilizará incrustaciones del modelo de incrustaciones de solo texto para los fragmentos de texto extraídos.",
```

Comment:
```suggestion
        "textEmbeddings": "Cuando se selecciona, la búsqueda utilizará embeddings de texto para los fragmentos extraídos.",
```

Reply:
Ahh so embarassing, incrustaciones in our translations! thanks for catching that.